README.md

```diff
@@ -46,6 +46,11 @@ func main() {
 }
 ```
 
+If using GitHub Enterprise, use `NewEnterpriseClient` :
```

Comment:
Please make a few changes:

- If using ... → If you are using ...
- \`NewEnterpriseClient\` → \[\`githubv4.NewEnterpriseClient\`\]\(https\://godoc.org/github.com/shurcooL/githubv4#NewEnterpriseClient\)
- Remove the space before the colon.
- Add a blank line after the colon.